### DISCUSSION TITLE
Live Server with  VS Code does not update HTML classes when using the --jit flag

### DISCUSSION THREAD

**Discussion by Aleks-Shelukhin** [tailwindlabs/tailwindcss#5878] [Help] [answered]
I decided to try JIT-mode. I run with the flags `-o tailwind.css --jit --purge index.html -w`. Compiled without errors. Surveillance is turned on. I start to write classes in index.html, Live Server updates the browser (the page reloads), in the CSS file the styles are updated, but in the browser changes are not applied. In the browser inspector, you can see that HTML does not update classes. Only when you manually refresh the page F5, then the classes are updated. 
If you run the compilation without the `--jit` flag, then everything works. and `--purge` and `--watch` I do my work and in the browser all classes are updated and everything works as it should. The problem is observed in different browsers and in incognito mode
Here's what else I did: I put a clean VS Code and only one Live Server plugin doesn't work either; put another Live Server from Tapio https://github.com/tapio/live-server - the same problem.
Please, friends, help, can anyone know how to solve this problem? 
Dear Tailwindcss developers, if you see this branch, pay attention to this problem and help me solve it

**Comment by NovusOpus** [comment, marked as answer]
Try: [Five Server (Live Server)](https://marketplace.visualstudio.com/items?itemName=yandeu.five-server), works for me when working with Tailwind CLI, It's Dev Server with Live Reload. (Maintained Fork of Live Server).

**Reply by Aleks-Shelukhin** [reply]
Thanks! It solved the problem!
